core/edit/edit_command_actions.py

```diff
@@ -88,26 +88,36 @@ def edit_action(m) -> EditAction:
 }
 
 
-def run_action_callback(action: EditAction):
-    action_type = action.type
-
-    if action_type in simple_action_callbacks:
-        callback = simple_action_callbacks[action_type]
-        callback()
-        return
-
-    match action_type:
-        case "insert":
-            assert isinstance(action, EditInsertAction)
-            actions.insert(action.text)
-
-        case "wrapWithDelimiterPair":
-            assert isinstance(action, EditWrapAction)
-            return lambda: actions.user.delimiter_pair_wrap_selection(action.pair)
-
-        case "applyFormatter":
-            assert isinstance(action, EditFormatAction)
-            actions.user.formatters_reformat_selection(action.formatters)
-
-        case _:
-            raise ValueError(f"Unknown edit action: {action_type}")
+@mod.action_class
+class Actions:
+    def run_action_callback(action: EditAction):
+        """
+        Run a callback that applies an edit action to the selected
+        Intended for internal use and overwriting
+        """
+        action_type = action.type
+
+        callback = actions.user.get_simple_action_callback(action_type)
+        if callback:
+            callback()
+            return
+
+        match action_type:
+            case "insert":
+                assert isinstance(action, EditInsertAction)
+                actions.insert(action.text)
+
+            case "wrapWithDelimiterPair":
+                assert isinstance(action, EditWrapAction)
+                return lambda: actions.user.delimiter_pair_wrap_selection(action.pair)
```

Comment:
This was not your code, but it looks like there is an error here in which a lambda is returned rather than just running the action. Could you fix this as part of your PR?

Reply:
Fixed, and also removed the reliance on string literals for the match